<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Banned</class>
 <widget class="QWidget" name="Banned">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>Drag &amp; Drop files containing SHA256 hashes.
You can Drag &amp; Drop binaries too.
It will create a list of hash to import in VMware Carbon Black Cloud</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_droparea" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_hashlist"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_save">
     <property name="text">
      <string>Save in txt file</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
